runtime/queue.c

```diff
@@ -775,16 +775,20 @@ qqueueTryLoadPersistedInfo(qqueue_t *pThis)
 	DEFiRet;
 	strm_t *psQIF = NULL;
 	struct stat stat_buf;
+	int errNbr;
 
 	ISOBJ_TYPE_assert(pThis, qqueue);
 
 	/* check if the file exists */
 	if(stat((char*) pThis->pszQIFNam, &stat_buf) == -1) {
-		if(errno == ENOENT) {
+		errNbr = errno;
+		if(errNbr == ENOENT) {
+			LogError(0, RS_RET_IO_ERROR, "queue: clean startup, no .qi file found");
```

Comment:
this is actually not an error, but only a debug status message. It is a normal state. This must not be exposed to the user.